site-book/README.md

```diff
@@ -0,0 +1,42 @@
+# Site Book Metron documentation 
+
+Metron's Site Book is an attempt at producing documentation that is
+
+-Versioned and reviewed,
```

Comment:
Probably want to put a space between -V and remove the comma at the end to get:
\- Versioned and reviewed